bin/docker-push

```diff
@@ -31,7 +31,11 @@ then
   exit 1
 fi
 
-docker tag -f "${DOCKERTAG}:latest" "${EXTERNAL_REGISTRY_BASE_DOMAIN}/${REPOSITORY_NAME}:${CI_REF}"
+DOCKER_TAG_COM="docker tag"
+docker version | grep -e Version | head -n 1 | grep  1.9 && DOCKER_TAG_COM="${DOCKER_TAG_COM} -f "
```

Comment:
So `DOCKER_TAG_COM` only gets converted from `"docker tag"` to `"docker tag -f"` if the `grep 1.9` succeeds, right? (Meaning the docker version *matches* `1.9` anywhere in that version string...?)

Are you looking specifically for version like 17.01.9 and below? I'm running 17.06.1-ce-rc1 currently on my local machine. I guess I'm asking about the `grep 1.9` command specifically and where "1.9" came from.